OctopusDSC/DSCResources/cTentacleAgent/cTentacleAgent.psm1

```diff
@@ -400,7 +402,13 @@ function Install-Tentacle
     Request-File $actualTentacleDownloadUrl $tentaclePath
 
     Write-Verbose "Installing MSI..."
-    if (-not (Test-Path "$TentacleHomeDirectory\logs")) { New-Item -type Directory "$TentacleHomeDirectory\logs" }
+    if(-not (Test-Path $env:TEMP))
+    {
+        Write-Verbose "Configured temp folder does not currently exist, creating..."
+        New-Item $env:TEMP -ItemType Directory -force # an edge case when the env var exists but the folder does not
+    }
+
+    if (-not (Test-Path "$TentacleHomeDirectory\logs")) { New-Item -type Directory "$TentacleHomeDirectory\logs" -force  }
```

Comment:
Why do we need the force here?
If its to create the parent directory if it doesn't exist, that already happens above?